### PR TITLE
python-simplepam-0.1.5; pyxtrlock-0.4

### PR DESCRIPTION
**Pull request by Nexus01**
python-simplepam: new, 0.1.5
pyxtrlock: new, 0.4

### CODE REVIEW

File: extra-python/pyxtrlock/autobuild/prepare [hunk 1]
```diff
@@ -0,0 +1,3 @@
+abinfo "Adding the code of utf-8 into setup.py"
```
Comment (MingcongBai) [inline extra-python/pyxtrlock/autobuild/prepare]: I don't think this `abinfo` is sufficiently accurate for what this does. Where did this routine come from (from you or from another distribution? If it's the latter, please cite this line in `abinfo`).

File: extra-python/pyxtrlock/autobuild/prepare [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+sed -i '1 i# -*- coding: utf-8 -*' "$SRCDIR"/setup.py
+alias python=python3
```
Comment (MingcongBai) [inline extra-python/pyxtrlock/autobuild/prepare]: Why is this necessary?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-python/pyxtrlock/autobuild/prepare (hunk 2 above)
Comment (Nexus01) [reply extra-python/pyxtrlock/autobuild/prepare]: Oh, I think the whole `prepare` file is unnecessary. I use this file to solve `SyntaxError: Non-ASCII character` when I didn't know `NOPYTHON2=1`. I will delete `prepare`.

**Comment by MingcongBai** [comment]
Closing - this is *not* a rejection.

We are switching to a topic-based maintenance system. Please re-create your PR against our `stable` branch, use our new Pull Request template, and work in accordance to our [maintenance guidelines](https://wiki.aosc.io/developer/packaging/topic-based-maintenance-guideline/).
